zombies.js

```diff
@@ -92,6 +122,14 @@
  * @name checkPack
  */
 
+  this.checkPack = function () {
```

Comment:
These methods down are not defined on the Class's prototype. They should be declared out the constructor and added to the Class's prototype.